src/Clock.tsx

```diff
@@ -0,0 +1,62 @@
+import { Component } from 'react';
+
+type Props = {
+  name: string,
+};
+
+type State = {
+  currentTime: Date,
+};
+
+export class Clock extends Component<Props, State> {
+  state: Readonly<State> = {
+    currentTime: new Date(),
+  };
+
+  timerId = 0;
+
+  componentDidMount() {
+    this.timerId = window.setInterval(() => {
+      const currentTime = new Date();
+
+      this.setState({ currentTime });
+
+      // eslint-disable-next-line
+      console.info(this.getTimeFromDate(currentTime));
+    }, 1000);
+  }
+
+  componentDidUpdate(prevProps: Readonly<Props>) {
+    const { name } = this.props;
+
```

Comment:
you need to add a some condition here so that the name output in the console is displayed correctly